CoreScriptsRoot/Modules/PlayerlistModule.lua

```diff
@@ -781,7 +782,27 @@ function popupHidden()
 end
 playerDropDown.HiddenSignal:connect(popupHidden)
 
+local function openPlatformProfileUI(rbxUid)
+	if not rbxUid or rbxUid < 1 then return end
+	pcall(function()
+		local platformService = game:GetService('PlatformService')
+		if platformService then
+			local platformId = platformService:GetPlatformId(rbxUid)
```

Comment:
GetPlatformId() returns a string, as the XDK API uses Platform::String^ for the xuid value. In addition, the PopupProfileUI() calls into an XDK function that expects Platform::String^.

Length is checked because the XDK function for opening the gamer card throws on empty string. The c++ function does account for this case, but I did an extra check here.